### ISSUE TITLE
Create sponsor team pictograms to use in detail pages

### ISSUE THREAD

**Issue by mattrosno** [carbon-design-system/carbon-platform#284]
Every sponsoring team has an icon that we use in the catalogs. The design spec has this at 64px in the library and asset detail page dashboards. I think we'll need an icon and pictogram per team.

<img width="1392" alt="image" src="https://user-images.githubusercontent.com/1691245/155606441-100f45d4-21e9-4f9b-9683-9ff01264afa6.png">

**Comment by kingtraceyj** [comment]
here are the pictograms we currently have. i'm waiting on brand approval for carbon for .com and carbon for products in the [brand repo](https://github.ibm.com/brand/Pictograms/issues/152).


[pictos.zip](https://github.com/carbon-design-system/carbon-platform/files/8439092/pictos.zip)

**Comment by mattrosno** [comment]
@kingtraceyj @jeanservaas here are the sponsor teams that we're using so far: https://github.com/carbon-design-system/carbon-platform/blob/main/services/web-app/data/teams.js

Questions:

1. Per that screenshot above, we have an "IBM.com" sponsor team with their own sponsor icon. I don't see that in pictos.zip. Are we consolidating so all of the Carbon for IBM.com libraries are now sponsored by the Carbon team (as a result of our 2022 re-org)? That means we'd no longer use the `Svg32CarbonForIbmDotcom` custom icon anymore.
2. We currently have a Carbon for IBM Products icon. In our mock indexing, we didn't apply that anywhere yet because we're using the Cloud, Watson, and Security icons as they sponsor the libraries, which gets rolled up into the Carbon for IBM Products PAL. These sponsor icons are at the library level, not the PAL (local system) level. Unless we identify a library that is sponsored at the PAL level, we don't need a Carbon for IBM Products pictogram?

**Comment by mattrosno** [comment]
If the answers are (1) Carbon team now sponsors IBM.com libraries, and (2) Carbon for IBM Products pictogram isn't needed, then I'll open a dev issue to temporarily use these SVGs directly.

**Comment by kingtraceyj** [comment]
here are the .com and the products pictograms:

[DotCom_Product.zip](https://github.com/carbon-design-system/carbon-platform/files/8491207/DotCom_Product.zip)

**Comment by kingtraceyj** [comment]
thoughts on the questions above:

1. We can use the pictogram I just attached for dot com libraries. I like the differentiation of the pictograms and that they are less tied to internal org structures. i realize we did do that with some of the carbon things, but it didn't make sense to pull those out. 
2. I think we might need it down the road although we don't need it now. I think the product teams do have some assets that aren't tied to watson, cloud, and security.
